cmd/update-operator/main.go

```diff
@@ -15,6 +15,8 @@ import (
 )
 
 var (
+	preRebootActions        flagutil.StringSliceFlag
```

Comment:
these are annotations, not "actions" (let's avoid this term in the real implementation, it doesn't mean much)